src/mlpack/methods/ann/layer/log_softmax_impl.hpp

```diff
@@ -33,7 +33,7 @@ void LogSoftMax<InputDataType, OutputDataType>::Forward(
   output = (maxInput - input);
 
   // Approximation of the hyperbolic tangent. The acuracy however is
-  // about 0.00001 lower as using tanh. Credits go to Leon Bottou.
+  // about 0.00001 lower as using tanh. Credits goes to Leon Bottou.
```

Comment:
Fixed in a0974a88488342e8e693a2d, for me "goes" sounds correct, but you are the expert 👍